src/main/java/entity/Stock.java

```diff
@@ -29,16 +29,16 @@ public String getIndustry() {
         return industry;
     }
 
-    public double getPrice() {
-        return price;
+    public double getMarketPrice() {
+        return marketPrice;
     }
 
     /**
      * Update the current market price of the stock
-     *
-     * @param price the new price
+     * This method will be called every two minutes. (interval)
```

Comment:
Delete this comment. We are not actually updating stocks every two minutes (as it is based on configuration). In additional `Stock` should not care about how its methods are invoked.